src/main/java/seedu/address/model/person/Picture.java

```diff
@@ -19,29 +20,42 @@
     // 10 MB
     public static final long MAX_FILE_SIZE = 10 * 1024 * 1024;
 
+    public static final Set<Byte[]> IMAGE_MAGIC_NUMBERS = new HashSet<>();
+
+    // Initialize magic numbers of images. null represents any value for 2 bytes.
+    static {
+        // png
+        IMAGE_MAGIC_NUMBERS.add(FileUtil.intArrayToByteArray(new int[] {0x89, 0x50, 0x4E, 0x47, 0x0D, 0x0A, 0x1A,
+                0x0A}));
+        // jpg, jpeg
```

Comment:
Differentiate the magic numbers for `jpg` and `jpeg`

Reply:
jpg and jpeg are the same file format. The different magic number are just subclass of the jpg formats (JFIF, EXIF, etc.)

Reply:
Ooo, the magic numbers for them are the same? I think better to write which subclass are u targeting for each magic number.

Reply:
Actually it's best to just remove anything after 3 bytes since most JPGs starts with `0xFFD8FF` anyways

Reply:
Yeap, u can do that too